src/tools/tidy/src/deps.rs

```diff
@@ -184,15 +184,15 @@ const EXCEPTIONS_RUSTC_PERF: ExceptionList = &[
 
 const EXCEPTIONS_RUSTBOOK: ExceptionList = &[
     // tidy-alphabetical-start
-    ("cssparser", "MPL-2.0"),
-    ("cssparser-macros", "MPL-2.0"),
-    ("dtoa-short", "MPL-2.0"),
-    ("encoding_rs", "(Apache-2.0 OR MIT) AND BSD-3-Clause"),
+    // ("cssparser", "MPL-2.0"),
+    // ("cssparser-macros", "MPL-2.0"),
+    // ("dtoa-short", "MPL-2.0"),
+    // ("encoding_rs", "(Apache-2.0 OR MIT) AND BSD-3-Clause"),
     ("mdbook", "MPL-2.0"),
-    ("ring", NON_STANDARD_LICENSE),
+    // ("ring", NON_STANDARD_LICENSE),
     ("ryu", "Apache-2.0 OR BSL-1.0"),
-    ("selectors", "MPL-2.0"),
-    ("subtle", "BSD-3-Clause"),
+    // ("selectors", "MPL-2.0"),
+    // ("subtle", "BSD-3-Clause"),
```

Comment:
Commented these out rather than actually removing them simply for testing; if this works correctly, I will remove them entirely.

The approach here assumes that we do *not* have `trpl` in `rustbook`’s `dependencies`, and therefore that none of these are direct dependencies of the tools shipped with Rust proper—only with the `trpl` crate.

Notably, then, `trpl` would be distributed with Rust *without* being in the workspace; do we need to address that by updating it *to* be included in the workspace and its dependencies checked?

Reply:
Fixed these, all is good here. 🎉